src/v2/mappers/TelegramGroupMapper.ts

```diff
@@ -25,4 +26,16 @@ export class TelegramGroupMapper {
       })),
     };
   }
+
+  getGroupsWithTelegramGroups (groups: DbGroupWithTelegramGroups[]) {
+    return {
+      groups: groups.map((telegramGroup) => ({
```

Comment:
rename the argument of the first map to group and the the argument of the second map to telegramGroup